scoap3/utils/arxiv.py

```diff
@@ -44,10 +44,12 @@ def get_clean_arXiv_id(record):
 
 
 def clean_arxiv(arxiv):
-    # drop 'arxiv:' prefix and version
+    # drop 'arxiv:' prefix, version and other information if there is
+    # also force ascii encoding and strip unnecessary characters
     if arxiv is None:
         return None
-    return arxiv.split(':')[-1].split('v')[0]
+
+    return arxiv.split(':')[-1].split('v')[0].split(' ')[0].encode('ascii', 'ignore').strip('"\'')
```

Comment:
The problem is that we received a package where the arXiv id contained some unicode characters. This should never happen since the id should only contain alphanumeric chars, slashes and dots, so here my goal is to emit all the unicode characters to extract the valid arxiv id.
https://arxiv.org/help/arxiv_identifier